components/api/README.md

```diff
@@ -4,10 +4,44 @@
 [![License: Apache-2.0](https://img.shields.io/github/license/exonum/exonum.svg)](https://github.com/exonum/exonum/blob/master/LICENSE)
 ![rust 1.36.0+ required](https://img.shields.io/badge/rust-1.36.0+-blue.svg?label=Required%20Rust)
 
-`exonum-api` provides a high-level wrapper around a web server,
-which uses  [`actix`] internally.
-The wrapper is used in [Rust services][rust-runtime] and in plugins
-for the Exonum node.
+`exonum-api` crate provides an extensible interface for building backend-agnostic
+HTTP APIs.
+
+Within Exonum, this crate is used by [Rust services][rust-runtime] and in
+plugins for the Exonum node.
+
+Under the hood `exonum-api` uses `actix`.
```

Comment:
Nit: It could make sense to link to `actix` or remove the link at the end of the readme.